config/newrelic.yml.erb

```diff
@@ -0,0 +1,41 @@
+
+<% unless ENV['MAS_ENVIRONMENT'] == 'qa' %>
+  common: &default_settings
+    # Required license key associated with your New Relic account.
+    license_key: <%= ENV['NEWRELIC_API_KEY'] %>
+
+    # Your application name. Renaming here affects where data displays in New
+    # Relic.  For more details, see https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/renaming-applications
+    app_name: Contento
+
+    # To disable the agent regardless of other settings, uncomment the following:
+    # agent_enabled: false
+
+    # Logging level for log/newrelic_agent.log
+    log_level: info
+
+
+  # Environment-specific settings are in this section.
+  # RAILS_ENV or RACK_ENV (as appropriate) is used to determine the environment.
+  # If your application has other named environments, configure them here.
+  development:
+    <<: *default_settings
+    app_name: Contento (Development)
+
+    # NOTE: There is substantial overhead when running in developer mode.
+    # Do not use for production or load testing.
+    developer_mode: true
+
+  test:
+    <<: *default_settings
+    # It doesn't make sense to report to New Relic from automated test runs.
+    monitor_mode: false
+
+  staging:
+    <<: *default_settings
+    app_name: Contento (Staging)
+
+  production:
+    <<: *default_settings
+    app_name: Contento (Production)
+<% end %>
```

Comment:
This PR needs to be closed. Essentially, we've set up new relic for all the key environments within Puppet which outputs the correct yaml file. The CMS was originally configured via Puppet but not finished so Puppet does not deploy it the same way as the other projects. We have to manually deploy the CMS to production.

This is changing for the new Azure environment. The CMS has been set up correctly there. As a temporary fix we've "hot" pushed a seperate new relic yaml file for the CMS production environments as that's the only one that Puppet can't create since it doesn't deploy it.

Here are some further PRs for reference https://github.com/moneyadviceservice/scripts/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+new+relic+